### DISCUSSION TITLE
commit only when branch name starts with feature/*

### DISCUSSION THREAD

**Discussion by JinoArch** [community/community#35519] [Repositories]
Wonder if we have an option in GitHub where developers can commit and push to a repo, only If the branch name starts with feature/*.

like 
if branch name is feature/* - developers are good to go  (push to repo)
else - not allowed to push to repo

pre-commit hooks can be skipped. Want to know if we have any other ways for the same.

Thanks

**Comment by DSDmark** [comment]
I don't know about that, but you can set default branch name if you want 

> inside setting and **Code, planning, and automation** section  
![Screenshot from 2022-10-08 15-14-42](https://user-images.githubusercontent.com/85744816/194701295-e854854f-28d6-4a3c-b66e-23c5960a3863.png)
